cpp/data/frameProcessor/src/XspressProcessPlugin.cpp

```diff
@@ -220,8 +223,14 @@ void XspressProcessPlugin::configure(OdinData::IpcMessage& config, OdinData::Ipc
                         boost::to_string(this->frames_per_block_),
                         buffer.GetString());
 
-    LOG4CXX_INFO(logger_, "Number of frames per block set to " << this->frames_per_block_);
-  }
+      LOG4CXX_INFO(logger_, "Number of frames per block set to " << this->frames_per_block_);
+    }
+
+    if(config.has_param(XspressProcessPlugin::CONFIG_OFFSET))
+    {
+      this->offset = config.get_param<uint32_t>(XspressProcessPlugin::CONFIG_OFFSET);
+      LOG4CXX_INFO(logger_,"configured offset: " << offset);
+    }
```

Comment:
This should also be returned in `requestConfiguration` to provide a readback.

Reply:
Also I think the indentation is 4 instead of 2

Reply:
Just added the offset value to the reply of `requestConfiguration`. But I'm a bit confused about the indentation, do you mean just this instance needs the 4 spaces or the whole code needs to be indented by 4?

Reply:
These added lines are indented by 4 but should be 2 to match the surrounding lines

Reply:
Got it!
It's done now!